cli/ast/mod.rs

```diff
@@ -350,12 +350,23 @@ pub fn transpile_module(
   options: &EmitOptions,
   cm: Rc<SourceMap>,
 ) -> Result<(Rc<deno_ast::swc::common::SourceFile>, Module), AnyError> {
-  let source = strip_bom(source);
+  let source = if media_type == MediaType::Json {
+    format!(
+      "export default JSON.parse(`{}`);",
+      source.replace("${", "\\${")
```

Comment:
Also should escape backtick?

It's funny, in dnt when implementing this feature I forgot the opposite (now fixed).